<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogmenu</class>
 <widget class="QDialog" name="Dialogmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>植此青绿小游戏</string>
  </property>
  <widget class="QGroupBox" name="groupBox14">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>91</y>
     <width>541</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <widget class="QPushButton" name="pushButton3">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <spacer name="horizontalSpacer_19">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>8</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_13">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>8</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton1">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>思源黑体 CN Medium</family>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <spacer name="horizontalSpacer_20">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>6</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="7">
     <widget class="QPushButton" name="pushButton4">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton2">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_18">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_12">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>6</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox58">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>162</y>
     <width>541</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_14">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton5">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_15">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton6">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_21">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton7">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_22">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton8">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_23">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox910">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>230</y>
     <width>541</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <spacer name="horizontalSpacer_27">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_28">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton9">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_29">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton10">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>55</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>江城正义体 400W</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_30">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <spacer name="horizontalSpacer_31">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>9</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelbackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>324</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>324</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>324</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labeltitle">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnranking">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>30</y>
     <width>121</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>55</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Medium</family>
     <pointsize>15</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnendlessmode">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>30</y>
     <width>121</width>
     <height>55</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>55</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Medium</family>
     <pointsize>15</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnback">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>0</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btntitle">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>17</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>labelbackground</zorder>
  <zorder>groupBox14</zorder>
  <zorder>groupBox58</zorder>
  <zorder>groupBox910</zorder>
  <zorder>labeltitle</zorder>
  <zorder>btnranking</zorder>
  <zorder>btnendlessmode</zorder>
  <zorder>btnback</zorder>
  <zorder>btntitle</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
